portal/models/app_text.py

```diff
@@ -31,6 +31,7 @@ def time_request(url):
     """Wrap the requests.get(url) and log the timing"""
     start = timeit.default_timer()
     response = requests.get(url)
+    response.raise_for_status()
```

Comment:
I like this idea, but I think you'll need to alter what exception types are caught by the clients, no?

Reply:
I was going to check the build and update it to what I found ("remote debugging"?). The build looks like exception Inception. I'll try reproducing locally...